style.css

```diff
@@ -110,4 +103,42 @@ body.maxCanvas details {
 
 body.maxCanvas #hydraCanvas.moveToTop {
   z-index: 100;
+}
+
+
+/* blockly stlyles */
+.blocklyContextMenu *,
+.blocklyTooltipDiv,
+.blocklyDropDownDiv {
+  text-shadow: none !important;
+}
+
+.blocklyScrollbarVertical{
+  display: none;
+}
+
+.blocklyScrollbarHorizontal{
```

Comment:
we might need it for mouse users? trackpad can handle xy scroll, but mouse can only do y, that too if it has scroll wheel. maybe can style it so it blends with background a little more.

Reply:
Scroll Wheel/Trackpad xy scroll seems to be not working although, click and drag interaction inside blockly canvas to pan/scroll appears to work & works with mouse as-well, tested on windows too.